core/templates/pages/contributor-dashboard-page/services/translate-text-backend-api.service.ts

```diff
@@ -65,7 +65,12 @@ export class TranslateTextBackendApiService {
     const body = new FormData();
     body.append('payload', JSON.stringify(postData));
     imagesData.forEach(obj => {
-      const imageBlob = <Blob>obj.imageBlob;
+      let imageBlob: string | Blob;
+      if (obj.imageBlob !== null) {
+        imageBlob = obj.imageBlob;
+      } else {
+        throw new Error('No Image Data Found');
+      }
```

Comment:
Do we need to instantiate the imageBlob? Can we just access it directly from obj?
```suggestion
      if (obj.imageBlob === null) {
        throw new Error('No Image Data Found');
      }
```

Reply:
Done